ush/forecast_postdet.sh

```diff
@@ -191,6 +191,15 @@ EOF
     IEMS=${IEMS:-1}
   fi
 
+# NoahMP table
+      local noahmptablefile="${HOMEgfs}/parm/ufs/noahmptable.tbl"
+    if [[ ! -f ${noahmptablefile} ]]; then
+      echo "FATAL ERROR: missing noahmp table file ${noahmptablefile}"
+      exit 1
+    else
+      ${NLN} "${noahmptablefile}" "${DATA}/noahmptable.tbl"
+    fi
```

Comment:
Please correct the indentation.

```suggestion
  # NoahMP table
  local noahmptablefile="${HOMEgfs}/parm/ufs/noahmptable.tbl"
  if [[ ! -f ${noahmptablefile} ]]; then
    echo "FATAL ERROR: missing noahmp table file ${noahmptablefile}"
    exit 1
  else
    ${NLN} "${noahmptablefile}" "${DATA}/noahmptable.tbl"
  fi
```